cardano-node/src/Cardano/Node/Configuration/POM.hs

```diff
@@ -407,7 +407,6 @@ instance FromJSON PartialNodeConfiguration where
               Just True  -> Last $ Just EnabledP2PMode
 
       -- Peer Sharing
-      -- DISABLED BY DEFAULT
```

Comment:
Instead of removing it, I would have written "ENABLED BY DEFAULT".

Reply:
That seems fine to guide the reader, but the haddocks themselves do not mention the default value. For example:
![image](https://github.com/user-attachments/assets/c40f4e39-6117-4f18-b04c-3aa1ab3ef128)

Reply:
Aha, good point. I agree